.circleci/config.yml

```diff
@@ -316,14 +320,17 @@ references:
             command: |
                 USER=$(echo $CONTRIB_BRANCH | cut -d ":" -f 1)
                 BRANCH=$(echo $CONTRIB_BRANCH | cut -d ":" -f 2)
-                PACK=$(echo $CONTRIB_BRANCH | cut -d ":" -f 3)
-                echo 'Copy the changes from the contributor branch $USER/$BRANCH in the pack $PACK'
-                git remote add $USER git@github.com:$USER/content.git
-                git fetch $USER $BRANCH
-                rm -rf $PACK
-                git checkout $USER/$BRANCH $PACK
-                exit 0
-
+                python3 ./Utils/get_contrib_pack.py -p $PULL_REQUEST_NUMBER -b $BRANCH -u $USER
+  notify_contrib_pr: &notify_contrib_pr
+    when:
+      condition: << pipeline.parameters.contrib_branch >>
+      steps:
+        - run:
+            name: Comment on the contribution Pull Request
+            when: always
+            command: |
+              SERVER_URL=$(jq -r 'select(.[].Role == "Demisto Marketplace") | .[].InstanceDNS' ./env_results.json)
```

Comment:
add comment here that `Demisto Marketplace` is here to in order to refer to server master